src/main/java/mServer/crawler/sender/arte/MediathekArte_de.java

```diff
@@ -46,10 +46,29 @@
 
 public class MediathekArte_de extends MediathekReader
 {
+    /*
+     * Informationen zu den ARTE-URLs:
+     * {} sind nur Makierungen, dass es Platzhalter sind, sie gehören nicht zur URL.
+     * 
+     * Allgemeine URL eines Films:  (050169-002-A = ID des Films); (die-spur-der-steine = Titel)
+     * http://www.arte.tv/de/videos/{050169-002-A}/{die-spur-der-steine}
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)